src/main/scala/mesosphere/marathon/api/v2/json/Formats.scala

```diff
@@ -782,3 +784,16 @@ trait V2Formats {
     (__ \ "version").formatNullable[Timestamp].withDefault(Group.defaultVersion)
   )(Group(_, _, _, _, _), unlift(Group.unapply))
 }
+
+trait PluginFormats {
+
+  implicit lazy val pluginDefinitionFormat: Writes[PluginDefinition] = (
+    (__ \ "id").write[String] ~
+    (__ \ "plugin").write[String] ~
+    (__ \ "implementation").write[String] ~
+    (__ \ "tags").writeNullable[Set[String]] ~
```

Comment:
What's the goal of this meta data? It is not available to the plugins nor used by Marathon itself.

Is it to work around not being able to write comments in the configuration file? If that's the case, isn't `info` enough?